control_service/requirements.txt

```diff
@@ -1,8 +1,10 @@
-flask
-werkzeug
 sqlalchemy
-flask_sqlalchemy
-jwt
 schema
-Flask-Caching
-psycopg2-binary
+fastapi
+databases
+aiosqlite
+passlib[bcrypt]
+python-multipart
+pyjwt
+aiodns
+postgresql
```

Comment:
There is no package `postgresql` (according to *pip* on `docker-compse build`). Did you mean `py-postgresql`?

Reply:
done